server/utils/premises/premisesActions.test.ts

```diff
@@ -119,4 +119,24 @@ describe('premisesActions', () => {
       })
     })
   })
+
+  describe('for users with no role', () => {
+    const user = userDetails.build({ roles: [] })
+    const premises = premisesFactory.build()
+
+    it('includes only the manage beds and manage out of service bed actions', () => {
```

Comment:
I think the problem with asserting the equality of the whole list of links is that when something changes we don't get a useful error. That's why we introduced the `toContainManageAction()` and `toContainMenuItem()` matchers.

I think what we actually want to assert here is that `premisesActions()` includes the new link to "Manage out of service beds"?

It's great to describe the fact that we're also showing the "Manage beds" to users with roles but I think if we describe this in 2 scenarios it will be clearer and much easier to maintain e.g.

```ts
describe('for users with no role', () => {
   it('includes the "manage beds" action', () => {})
   it('includes the "out of service beds" action', () => {})
 })
```

Reply:
right, yes! I see now, thanks. have implemented the change 👍